src/cpp/fastdds/publisher/DataWriterImpl.cpp

```diff
@@ -435,7 +436,7 @@ bool DataWriterImpl::remove_min_seq_change()
     return history_.removeMinChange();
 }
 
-bool DataWriterImpl::remove_all_change(
+bool DataWriterImpl::remove_all_changes(
```

Comment:
Change if the method's name finally changes to `clear_history` as proposed elsewhere